components/raftstore/src/store/peer.rs

```diff
@@ -478,7 +484,7 @@ where
 
     proposals: ProposalQueue<EK::Snapshot>,
     leader_missing_time: Option<Instant>,
-    leader_lease: Lease,
+    pub leader_lease: Lease,
```

Comment:
How about using an access method?

Reply:
Other fields are public, I think no bother to add an access method for it only.

Reply:
If it's just for read access, it's still necessary. For example, I have found it very convenient multiple times that knowing `proposals` and `pending_reads` are private so I only need to check the single file for potential pitfalls.